target/linux/ramips/dts/WD03.dts

```diff
@@ -0,0 +1,130 @@
+/dts-v1/;
+
+#include "mt7620a.dtsi"
+
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "WD03", "ralink,mt7620a-soc";
+	model = "WD03";
+
+	chosen {
+		bootargs = "console=ttyS0,115200";
+	};
+
+	gpio-leds {
+		compatible = "gpio-leds";
+
+		green-wifi {
+			label = "wd03:green:wifi";
+			gpios = <&gpio2 0 1>;
+		};
+
+
+		blue-wifi {
+			label = "wd03:blue:wifi";
+			gpios = <&gpio3 0 1>;
+		};
+
+	};
+
+	gpio-keys-polled {
+		compatible = "gpio-keys-polled";
+		#address-cells = <1>;
+		#size-cells = <0>;
+		poll-interval = <20>;
+		reset {
+			label = "reset";
+			gpios = <&gpio2 1 1>;
+			linux,code = <KEY_RESTART>;
+		};
+
+		KEY_EJECTCD {
+			label = "sdcard";
+			gpios = <&gpio2 2 1>;
+			linux,code = <KEY_EJECTCD>;
+		};
+
+	};
+};
+
+&gpio0 {
+	status = "okay";
+};
+
+&gpio1 {
+	status = "okay";
+};
+
+&gpio2 {
+	status = "okay";
+};
+		
+&gpio3 {
+	status = "okay";
+};
+
+&spi0 {
+	status = "okay";
+
+	m25p80@0 {
+		#address-cells = <1>;
+		#size-cells = <1>;
+		compatible = "jedec,spi-nor";
+		reg = <0>;
+		spi-max-frequency = <10000000>;
+
+		partition@0 {
+			label = "u-boot";
+			reg = <0x0 0x30000>;
+		};
+
+		partition@30000 {
+			label = "u-boot-env";
+			reg = <0x30000 0x10000>;
+			read-only;
+		};
+
+		factory: partition@40000 {
+			label = "factory";
+			reg = <0x40000 0x10000>;
+			read-only;
+		};
+
+		partition@50000 {
+			label = "firmware";
+			reg = <0x50000 0x7b0000>;
+		};
+
+		partition@ff0000 {
+			label = "art";
+			reg = <0xff0000 0x10000>;
```

Comment:
copy paste leftover? I don't see any reference to this partition and it is at offset 15,93MB of the 8 MB flash chip.